TicTacToe2/TicTacToeGame.cs

```diff
@@ -24,115 +28,251 @@ public void Play()
             }
         }
 
+        private void ChooseBoard()
```

Comment:
This function works, but we're doing something very similar to GetUserInput(). There's an action common across both functions - getting numeric input from the user in a certain range. It may be worth creating a separate function to do that.

The function declaration may looks something like

```c#
int GetNumericInput(int min, int max)
```